WORKSPACE

```diff
@@ -6,7 +6,7 @@ workspace(name = "org_pubref_grpc_greetertimer")
 # repo directly.
 git_repository(
     name = "io_bazel_rules_dotnet",
-    commit = "f8950cbf9456df79920514325c17139355d13671",
+    commit = "1a6ca96fe05bca83782464453ac4657fb8ed8379",
```

Comment:
Fixes this error:

```
ERROR: /tmpdir/external/io_bazel_rules_dotnet/dotnet/csharp.bzl:25:10: name 'set' is not defined
```

This is the version used by the current rules_protobuf master:
https://github.com/pubref/rules_protobuf/blob/a58399a5c8e2908630eb65c31a1a8e2a3e320044/WORKSPACE#L46

Note that we still can't use the latest rules_dotnet because dotnet/csharp.bzl was removed in:
https://github.com/tomaszstrejczek/rules_dotnet/pull/3